src/ramanchada2/io/experimental/read_txt.py

```diff
@@ -35,6 +36,11 @@ def read_txt(data_in: TextIO) -> Tuple[NDArray, NDArray, Dict]:
         intensities = data['Processed_Data'].to_numpy()
         meta['@axes'] = ['Raman_Shift']
         meta['@signal'] = 'Processed_Data'
+    elif lines[0].startswith('DeviceSN\t'):
+        data, meta = lightnovo_tsv(lines)
+        positions = data['Position'].to_numpy()
+        intensities = data['Amplitude'].to_numpy()
+        meta['@signal'] = 'DarkSubtracted'
```

Comment:
It's not necessarily DarkSubtracted - in fact the files we have are not dark subtracted